### DISCUSSION TITLE
Meta Quest VR Integration for Teleop

### DISCUSSION THREAD

**Discussion by MoreTore** [isaac-sim/IsaacLab#997] [General]
Would anyone be interested in adding support for Meta Quest for teleoperation? I have seen demos of large companies using this for training but i can't find any implementations of this. I think IsaacLab would be a good fit for this.

I see some discussion about this on the forum https://forums.developer.nvidia.com/t/integration-between-isaacsim-and-create-xr/229108/8
but I would like an out-of-the-box example that maps the controller and head unit poses to robot end affecters.

**Comment by Dhoeller19** [comment]
Hi @MoreTore,
This is a great suggestion. We are ramping up the efforts internally to support VR for teleoperation and I expect we should have something beginning of next year.

**Comment by brukg** [comment]
@MoreTore @Dhoeller19 check this pull request it works with meta quest using ALVR and steamvr the pull request is still open 
- https://github.com/isaac-sim/IsaacLab/pull/4419
